rvasp.go

```diff
@@ -257,7 +257,7 @@ func (s *Server) simulateTRISA(stream pb.TRISADemo_LiveUpdatesServer, req *pb.Co
 
 	// Lookup the wallet of the beneficiary
 	var beneficiary Wallet
-	if err = s.db.Preload("Provider").Where("email = ?", transfer.Beneficiary).Or("address = ?", transfer.Beneficiary).First(&beneficiary).Error; err != nil {
```

Comment:
Note that beneficiary lookup was with both email and address - assuming the bug was with originator account lookup by wallet address or email.